lib/ansible/modules/cloud/amazon/aws_codebuild.py

```diff
@@ -115,11 +115,11 @@
     environment:
         description:
             - Information about the build environment for the build project.
-        required: true
+        required: false
```

Comment:
By convention, we only mention `required` when it's true.
```suggestion
```

Reply:
I'd gone with the what had been done in the rest of the file.  Updated this batch of files for consistency.